### ISSUE TITLE
No Code completion while typing

### ISSUE THREAD

**Issue by Chris2011** [pedrohidalgo/nbshell#2]
When I type "c" i expect that the code completion should comes up immediately but it doesn't. It only comes up when you hit ctrl + space. Please fix it and add a option to Options -> Editor -> code completion to set this as default or not. So if someone doesn't like that, they can deactivate it.

Regards

Chris

**Comment by Chris2011** [comment]
Anything new here?

**Comment by edemen** [comment]
I don't think this is a bug, as it works all the same to me in Java or PHP on Netbeans 8.2, and I think it is quite acceptable. I would not want code completion triggered on every character that I type.

**Comment by Chris2011** [comment]
Ok this is more a Feature request. But when you compare it with Java or PHP it is the same that there is an Option to have it everywhere or Not. And for me i want it right after i Type one Character. It doesn't make Sense in Strings but for Keywords, IT does. So the request is more for the Option to make it optional to have it while typing or while hitting Ctrl+space.
